book/website/history.md

```diff
@@ -0,0 +1,43 @@
+# History of The Turing Way project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
```

Comment:
Technically the funding came through in 2018 and then we launched at the SSI Collaborations Workshop in 2019.

Reply:
Adding this in - thought maybe it should be moved further up?